deltachat-server/src/bin/deltachat-server/main.rs

```diff
@@ -0,0 +1,70 @@
+use anyhow::Result;
+
+use std::path::PathBuf;
+use yerpc::{RpcClient, RpcSession};
+
+use deltachat_jsonrpc::api::events::event_to_json_rpc_notification;
+use deltachat_jsonrpc::api::{Accounts, CommandApi};
+
+use futures_lite::stream::StreamExt;
+use std::sync::Arc;
+use tokio::io::{self, AsyncBufReadExt, BufReader};
+use tokio::sync::RwLock;
+use tokio::task::JoinHandle;
```

Comment:
not very consistent grouping of the imports... usually it's grouped "std", "thirdparty" "self" (last one doesn't exist here).